_posts/clairewlj/2016-02-03-LingjiesChapter5Exercises.md

```diff
@@ -0,0 +1,13 @@
+---
+author: clairewlj
+layout: post
+title: "Lingjie's Chapter 5 Loop Exercises"
+---
+
+This is my loop exercise 1:
+<iframe src="https://trinket.io/embed/python/1ed86b3d87" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This is my loop exercise 2:
+<iframe src="https://trinket.io/embed/python/57dff0f321" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection: I've learned two things from these exercises. One is that the command 'print' can use commas as '+' only when there are no no parenthese, and that the data type everything input is string, so for the numbers we still need to convert them into integer or float types, or there'll be some problems when doing computation.
```

Comment:
Nice work.

The print command is a little strange on Trinket because we try to support python 2 and Python 3.  Apologies for that.  Once we're using server-side python things will be clearer.

One thing I noticed in your code: if the user presses enter without entering a number you get `nan`, 'not a number'. Can you think of how to deal with this edge case?